test/cluster/test_tablets_parallel_decommission.py

```diff
@@ -0,0 +1,449 @@
+#
+# Copyright (C) 2025-present ScyllaDB
+#
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+#
+import asyncio
+import logging
+import time
+
+import pytest
+
+from test.cluster.conftest import skip_mode
+from test.pylib.internal_types import ServerInfo
+from test.pylib.manager_client import ManagerClient
+from test.pylib.tablets import get_replica_count_by_host
+from test.pylib.util import wait_for
+
+logger = logging.getLogger(__name__)
+
+
+async def count_requests_queued(manager: ManagerClient, coord_srv: ServerInfo, request_type: str) -> int:
+    tasks = await manager.api.get_tasks(coord_srv.ip_addr, 'node_ops')
+    logger.info(f'tasks: {tasks}')
+    return len(list(filter(lambda t: t['type'] == request_type, tasks)))
+
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_tablets_are_drained_in_parallel(manager: ManagerClient):
+    """
+    Verifies that when we start decommissioning two nodes at the same time,
+    migrations draining tablets from both nodes are interleaved, indicating that
+    the decommissions are indeed progressing in parallel and not sequentially.
+    """
+
+    cfg = {'enable_tablets': True}
```

Comment:
It's unneeded now. We have `tablets_mode_for_new_keyspaces`, and it's set to true for cluster tests. The same goes for other test cases.